src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/PhpConfigReferenceDumpPass.php

```diff
@@ -34,6 +34,8 @@ class PhpConfigReferenceDumpPass implements CompilerPassInterface
 
         namespace Symfony\Component\DependencyInjection\Loader\Configurator;
 
+        use Symfony\Component\Config\Loader\ParamConfigurator;
```

Comment:
whats about aliasing to make the rest shorter?

```suggestion
        use Symfony\Component\Config\Loader\ParamConfigurator as param;
```

Reply:
Sure, done.